include/material/newtonian.tcc

```diff
@@ -31,7 +31,7 @@ Eigen::Matrix<double, 6, 1> mpm::Newtonian<2>::compute_stress(
 
   const unsigned phase = 0;
```

Comment:
@kks32 did you forget to delete this one?

Reply:
Yep!